test-configs.yaml

```diff
@@ -642,11 +647,26 @@ device_types:
     class: arm-dtb
     boot_method: uboot
 
+  imx7ulp-evk:
+    mach: imx
+    class: arm-dtb
+    boot_method: uboot
+
+  imx8mm-ddr4-evk:
```

Comment:
This board doesn't appear to be in linux-next, from `next-20200720`:
```
$ ls arch/arm64/boot/dts/freescale/*ddr4*
arch/arm64/boot/dts/freescale/imx8mn-ddr4-evk.dts
```
Only the `imx8mn-ddr4-evk` is there. Maybe `imx8mn-ddr4-evk` should be dropped for now?  It's not going to be tested in KernelCI until it's at least in linux-next.  Is it on its way to a maintainer branch?